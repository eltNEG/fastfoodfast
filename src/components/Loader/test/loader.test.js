import React from 'react';
import { shallow } from 'enzyme';
import RedirectLoader from '..';

describe('RedirectLoader', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<RedirectLoader />);
    expect(wrapper).toMatchSnapshot();
  });
});
