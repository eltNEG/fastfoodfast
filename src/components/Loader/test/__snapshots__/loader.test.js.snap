// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RedirectLoader should match snapshot 1`] = `
<div
  className="redirect-loader"
>
  <Loader
    active={true}
    inline="centered"
    size="massive"
  >
    Loading...
  </Loader>
</div>
`;
